tests/suite/regression.rs

```diff
@@ -850,8 +850,44 @@ fn regression_test_61() {
 
 #[test]
 fn regression_test_62() {
-    let original = "<foo";
```

Comment:
Where did this testcase go???

Reply:
It was added as part of #438 and should be retained, I think.

Reply:
Agreed, but it was somehow only removed from the *generated* tests. It looks like it was never added to the test source?

Reply:
Ah yes, looking at your PR that seems to be the case. I shall add it to the test source in this PR.

Reply:
Right, that's what happened, I added it to the wrong file. Thanks!